src/test/java/ClientFieldTest.java

```diff
@@ -117,11 +151,32 @@ void boolVector() {
         CaffaBooleanArrayField typedBoolArrayField = boolArrayField.cast(CaffaBooleanArrayField.class, Boolean.class);
         assertNotNull(typedBoolArrayField);
         ArrayList<Boolean> values = typedBoolArrayField.get();
+
         assertEquals(4, values.size());
         assertEquals(true, values.get(0));
         assertEquals(false, values.get(1));
         assertEquals(false, values.get(2));
         assertEquals(true, values.get(3));
+
+        {
+            ArrayList<Boolean> newValues = new ArrayList<Boolean>() {
+                {
+                    add(false);
+                    add(false);
+                    add(true);
+                }
+            };
```

Comment:
Fancy!

This simpler to understand (and shorter):
ArrayList<Boolean> newValues = new ArrayList<Boolean>();
newValues.add(false);